ampd/src/evm/mod.rs

```diff
@@ -1,50 +1,4 @@
-use std::fmt::Display;
-
-use enum_display_derive::Display;
-use serde::{Deserialize, Serialize};
-
 pub mod error;
 pub mod finalizer;
 pub mod json_rpc;
 pub mod verifier;
-
-#[derive(Debug, Deserialize, Serialize, PartialEq, Hash, Eq, Clone, Display)]
-pub enum ChainName {
-    Ethereum,
-    #[serde(untagged)]
-    Other(String),
-}
-
-impl PartialEq<connection_router_api::ChainName> for ChainName {
-    fn eq(&self, other: &connection_router_api::ChainName) -> bool {
-        self.to_string().eq_ignore_ascii_case(other.as_ref())
```

Comment:
Does the other definition of `ChainName` also compares ignoring cases?

Reply:
yes